### PR TITLE
routesrv: log route updates only when changed

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
* log and change route update metric only when routes changed
* speedup tests from 20s to 2.5s by reducing poll interval

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

File: routesrv/polling.go [hunk 1]
```diff
@@ -98,19 +92,20 @@ func (p *poller) poll(wg *sync.WaitGroup) {
...
-			routesBytes, initialized = p.b.formatAndSet(routes)
+			routesBytes, initialized, updated := p.b.formatAndSet(routes)
 			logger := log.WithFields(log.Fields{"count": routesCount, "bytes": routesBytes})
 			if initialized {
 				logger.Info(LogRoutesInitialized)
 				span.SetTag("routes.initialized", true)
 				routesInitialized.SetToCurrentTime()
-			} else {
+			}
+			if updated {
```
Comment (MustafaSaber) [inline routesrv/polling.go:105]: should we set SpanTag for `routes.updated`?

Comment (AlexanderYastrebov) [reply routesrv/polling.go:105]: Makes sense, thank you.
Added in the #2531

**Pull request merged**